README.md

```diff
@@ -22,6 +22,7 @@ If you'd like to help out, please see [CONTRIBUTING.md](CONTRIBUTING.md).
 [![Documentation](https://docs.rs/rustls/badge.svg)](https://docs.rs/rustls/)
 [![Chat](https://img.shields.io/discord/976380008299917365?logo=discord)](https://discord.gg/MCSB76RU96)
 [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/9034/badge)](https://www.bestpractices.dev/projects/9034)
+[![Gurubase](https://img.shields.io/badge/Gurubase-Ask%20Rustls%20Guru-006BFF)](https://gurubase.io/g/rustls)
```

Comment:
My personal opinion is that we should direct folks with questions to our issue tracker, or discord, and not to an LLM.

Both because it can't be relied upon to always deliver accurate answers and more crucially, because it doesn't feed back into the development process leading to improvements to our code and documentation.